### ISSUE TITLE
How to stash some full rejuvs

### ISSUE THREAD

**Issue by labrecquev** [kolton/d2bot-with-kolbot#984]
I'd like to stash full rejuvs with a maxquantity.
I tried to remove line 57 from town.js (the list of ignoredItemTypes, those who won't get stashed.
I then added the # # [maxquantity] == X tag on line 611 in Kolton.nip, For the quantity, i input double the amount stored on the belt (16 total, cause 8 spots on the belt).

Still it is not stashing rejuvs. Once my belt is full, it won't pick them up anymore.

**Comment by semirotta** [comment]
> 	Config.HPBuffer = 1; // Number of healing potions to keep in inventory.
	Config.MPBuffer = 1; // Number of mana potions to keep in inventory.
	Config.RejuvBuffer = 5; // Number of rejuvenation potions to keep in inventory.

**Comment by labrecquev** [comment]
> Oh never mind you mean like storaging them into stash. I think you need to use Kolton.nip
> Something like:
> [Name] == FullRejuvenationPotion # # [MaxQuantity] == 10

I've already tried to add the maxquantity tag after the fullrejuvenation line (line 611) in kolton.nip and it does not work.
I'll try copying yours above, right under the // MISC section

I've also tried to increase the buffer to keep in inventory. I doubt it will stash them though.

It worked. I don't know if it's the 2nd " [Name] == FullRejuvenationPotion # # [MaxQuantity] == 10 " line
or the increase of the Config.RejuvBuffer = 8; (orginally == 1)
